packages/discord.js/src/managers/ApplicationEmojiManager.js

```diff
@@ -65,12 +65,12 @@ class ApplicationEmojiManager extends CachedManager {
    * @returns {Promise<ApplicationEmoji|Collection<Snowflake, ApplicationEmoji>>}
    * @example
    * // Fetch all emojis from the application
-   * message.application.emojis.fetch()
+   * message.client.application.emojis.fetch()
```

Comment:
This should be consistent with the `create` method examples. Either use `client.application` or just `application` on both